test/e2e/seeder/contract-address-registry.js

```diff
@@ -1,8 +1,8 @@
 /*
  * Use this class to store pre-deployed smart contract addresses of the contracts deployed to
- * a local blockchain instance ran by Ganache.
+ * a local blockchain instance.
  */
-class GanacheContractAddressRegistry {
+class ContractAddressRegistry {
```

Comment:
just a rename, as now it's used both with ganache and anvil - no functionality change